packages/template-retail-react-app/app/pages/checkout/index.jsx

```diff
@@ -5,37 +5,55 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import React, {useEffect, useState} from 'react'
-import {FormattedMessage} from 'react-intl'
+import {FormattedMessage, useIntl} from 'react-intl'
 import {Alert, AlertIcon, Box, Button, Container, Grid, GridItem, Stack} from '@chakra-ui/react'
 import useNavigation from '../../hooks/use-navigation'
 import {CheckoutProvider, useCheckout} from './util/checkout-context'
 import ContactInfo from './partials/contact-info'
 import ShippingAddress from './partials/shipping-address'
 import ShippingOptions from './partials/shipping-options'
-// import Payment from './partials/payment'
+import Payment from './partials/payment'
 import OrderSummary from '../../components/order-summary'
 import {useCurrentCustomer} from '../../hooks/use-current-customer'
 import {useCurrentBasket} from '../../hooks/use-current-basket'
 import CheckoutSkeleton from './partials/checkout-skeleton'
+import {useUsid, useShopperOrdersMutation} from 'commerce-sdk-react-preview'
 
 const Checkout = () => {
+    const {formatMessage} = useIntl()
     const navigate = useNavigation()
-    const {globalError, step, placeOrder} = useCheckout()
+    const usid = useUsid()
+    const {step} = useCheckout()
+    const [error, setError] = useState()
     const {data: basket} = useCurrentBasket()
     const [isLoading, setIsLoading] = useState(false)
-    // Scroll to the top when we get a global error
+    const {mutateAsync: createOrder} = useShopperOrdersMutation('createOrder')
```

Comment:
Just observing that we're inconsistent in across the project with using mutation hooks. I've seen at least all of these styles:
```js
const createOrder = useShopperOrdersMutation('createOrder'); createOrder.mutate(...);
const createOrderMutation = useShopperOrdersMutation('createOrder'); createOrderMutation.mutate(...);
const {mutate} = useShopperOrdersMutation('createOrder'); mutate(...);
```

Eventually, we might want to try to be consistent. Or maybe just not bother...

Reply:
i've added a reminder to revisit this on my slack...